### ISSUE TITLE
Opening and closing of Tab-Manager-Plus

### ISSUE THREAD

**Issue by Applemann** [stefanXO/Tab-Manager-Plus#36]
When I don't have enabled `Open in own Tab by default` option and press shortcut for open Tab-Manager-Plus so opening the small popup window is quite slow and when I press it again then so it is closed.

When I have enabled `Open in own Tab by default` and press shortcut for open Tab-Manager-Plus so I am switched into my opened window immediately but when I press it again so it don't move me back into my last Tab.

Can you please solve opening the popup window speed or modify the shortcut for return back into my last Tab?

**Comment by mastef** [comment]
@Applemann 

1. Popup opening speed : how many tabs/windows do you have open?
2. Shortcut to go back to last opened tab - that's a good idea, I'll schedule that

**Comment by Applemann** [comment]
I have opened 65 tabs in 6 windows and sometimes it is opened in 1-2 second and sometimes in 6s

It looks like here: https://user-images.githubusercontent.com/2686906/68762921-74188580-0617-11ea-8452-cde64b512c66.png

**Comment by mastef** [comment]
@Applemann I have 120+ tabs open in Chrome and just opened an instance of Firefox with 120+ tabs. In the background I have some other annoying process running my CPU at 100%. The popups in both cases open within 1 second.

I just want to make 100% sure - are you running Tab Manager at version 5.1.3? I guess you are, since you have the huge icons. Just didn't see the "save window for later" button enabled.

I recall some slow opening times, but there have been many improvements in the past few updates. However I've just restarted my system. Maybe that helps? Restarting your browser? Just trying to reproduce the case here. 🤔

**Comment by Applemann** [comment]
Yes I have version 5.1.3 and after restart my browser it looks like better. Nothing is slow right now. 
I will close this issue tomorrow when I will not find any other problem with it.

Thank you very much for your fast response ;)

**Comment by Applemann** [comment]
Hello, after a few days later now without restarting or closing the browser window or operating system (I am using computer sleeping)  is the small window opening already slow again.

(This is only for your information if you can somehow fix it otherwise I will restart the browser or use Tab-Manager-Plus in separate tab.)

**Comment by mastef** [comment]
@Applemann thanks for letting me know. Maybe the browser doesn't clean up the resources correctly. I will add further clean ups to the extension and see if that helps.
